automation/src/test/scala/org/broadinstitute/dsde/test/api/SnapshotAPISpec.scala

```diff
@@ -314,6 +355,11 @@ class SnapshotAPISpec extends AnyFreeSpecLike with Matchers with BeforeAndAfterA
       content = payload)
   }
 
+  private def deleteSnapshotReference(projectName: String, workspaceName: String, resourceId: String)(implicit authToken: AuthToken) = {
+    val targetRawlsUrl  = Uri(Rawls.url).withPath(Path(s"/api/workspaces/$projectName/$workspaceName/snapshots/v2/$resourceId"))
+    Rawls.deleteRequest(uri = targetRawlsUrl.toString())
+  }
```

Comment:
This actually ended up not being used but I would advocate for keeping it here since future tests will probably want to test deleting a snapshot reference.